main.go

```diff
@@ -70,30 +72,38 @@ func loggingMiddleware(next http.Handler) http.Handler {
 }
 
 func main() {
-	var config Config
-	err := envconfig.Process("", &config)
-	if err != nil {
-		println("Unable to parse config " + err.Error())
-		os.Exit(1)
-	}
-	rubberneck.Print(config)
-
-	serverRoot, err := fs.Sub(uiFS, "ui")
-	if err != nil {
-		println("Unable to read ui filesystem ", err.Error())
-		os.Exit(1)
-	}
-
-	uiFs := http.FileServer(http.FS(serverRoot))
-	router := mux.NewRouter()
-	router.Use(loggingMiddleware)
+	kong.Parse(&Config)
 
-	router.HandleFunc("/nameifier/{seed}/{count}", nameHandler).Methods("GET")
-	router.HandleFunc("/nameifier/{seed}", blankHandler).Methods("GET")
-	router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
-	router.HandleFunc("/nameifier/", blankHandler).Methods("GET")
-	router.PathPrefix("/").Handler(uiFs)
+	// By default start server
```

Comment:
We should explicitly add a `--cli` flag or similar that we switch on so there's not any magic.